src/ReactiveSocket.cpp

```diff
@@ -151,6 +157,14 @@ bool ReactiveSocket::createResponder(
       handler_->handleFireAndForgetRequest(std::move(frame.data_));
       break;
     }
+    case FrameType::METADATA_PUSH: {
```

Comment:
If I move this part to ConnectionAutomation, then how can I get handler in that file?